content/de/docs/concepts/overview/what-is-kubernetes.md

```diff
@@ -0,0 +1,176 @@
+---
+
+{{% capture overview %}}
+Diese Seite ist eine Übersicht über Kubernetes.
+{{% /capture %}}
+
+{{% capture body %}}
+
+Kubernetes ist eine portable, erweiterbare Open-Source-Plattform zur Verwaltung von
+containerisierte Arbeitslasten und Services, die sowohl die deklarative Konfiguration als auch die Automatisierung erleichtert. 
+Es hat einen großes, schnell wachsendes Ökosystem. Kubernetes Dienstleistungen, Support und Tools sind weit verbreitet.
+
+Google hat das Kubernetes-Projekt 2014 als Open-Source-Projekt zur Verfügung gestellt. Kubernetes baut auf anderthalb Jahrzehnten 
+Erfahrung auf, die Google mit der Ausführung von Produktions-Workloads in großem Maßstab hat, kombiniert mit den besten Ideen und Praktiken der Community.
+
+## Warum brauche ich Kubernetes und was kann ich damit tun?
+
+Kubernetes hat eine Reihe von Funktionen. Es kann gesehen werden als:
+
+- eine Containerplattform
+- eine Microservices-Plattform
+- eine portable Cloud-Plattform
+und vieles mehr.
+
+Kubernetes bietet eine **containerzentrierte** Managementumgebung. Es koordiniert die Computer-, Netzwerk- und Speicherinfrastruktur 
+im Namen der Benutzer-Workloads. Dies bietet einen Großteil der Einfachheit von Platform as a Service (PaaS) mit der Flexibilität 
+von Infrastructure as a Service (IaaS) und ermöglicht die Portabilität zwischen Infrastrukturanbietern.
+
+## Wie ist Kubernetes eine Plattform?
+
+Auch wenn Kubernetes eine Menge Funktionalität bietet, gibt es immer wieder neue Szenarien, 
+die von neuen Funktionen profitieren würden. Anwendungsspezifische Workflows können optimiert werden, 
+um die Entwicklungsgeschwindigkeit zu beschleunigen. 
+Eine zunächst akzeptable Ad-hoc-Orchestrierung erfordert oft eine robuste Automatisierung in großem Maßstab. 
+Aus diesem Grund wurde Kubernetes auch als Plattform für den Aufbau eines Ökosystems von Komponenten und Tools 
+konzipiert, um die Bereitstellung, Skalierung und Verwaltung von Anwendungen zu erleichtern.
+
+[Labels](/docs/concepts/overview/working-with-objects/labels/) ermöglichen es den Benutzern, ihre Ressourcen 
+nach Belieben zu organisieren. [Anmerkungen](/docs/concepts/overview/working-with-objects/annotations/) ermöglichen es Benutzern,
+Ressourcen mit benutzerdefinierten Informationen zu versehen, um ihre Arbeitsabläufe zu vereinfachen und eine einfache Möglichkeit
+für Managementtools zu bieten, den Status von Kontrollpunkten zu ermitteln.
+
+
+Darüber hinaus basiert die [Kubernetes-Steuerungsebene](/docs/concepts/overview/components/) auf den gleichen APIs,
+die Entwicklern und Anwendern zur Verfügung stehen. Benutzer können ihre eigenen Controller, wie z.B. 
+[Scheduler](https://github.com/kubernetes/community/blob/{{< param "githubbranch" >}}/contributors/devel/scheduler.md), mit 
+ihren [eigenen APIs](](/docs/concepts/api-extension/custom-resources/) schreiben, die von einem 
+universellen [Befehlszeilenwerkzeug](/docs/user-guide/kubectl-overview/) angesprochen werden können.
```

Comment:
Weiter unten wird kubectl "Kommandozeilen-Tool" genannt. Vielleicht sollte das einheitlich sein

Reply:
Indeed. Changed it to "Kommandozeilen-Tool"